ci/ci-util.js

```diff
@@ -0,0 +1,48 @@
+var ncp = require('child_process');
+var fs = require("fs");
+var path = require('path');
+
+//------------------------------------------------------------------------------
+// global paths
+//------------------------------------------------------------------------------
+
+var commonPackagesSourcePath = path.join(__dirname, '..', 'common-npm-packages');
+exports.commonPackagesSourcePath = commonPackagesSourcePath;
+
+//------------------------------------------------------------------------------
+// generic functions
+//------------------------------------------------------------------------------
+
+var getDirectories = source =>
+  fs.readdirSync(source, { withFileTypes: true })
+    .filter(dirent => dirent.isDirectory())
+    .map(dirent => dirent.name)
+exports.getDirectories = getDirectories;
+
+var run = function (cl, inheritStreams) {
```

Comment:
[suggestion]
Can we add jsdoc with small explanation for utils?

something like: https://github.com/microsoft/azure-pipelines-tasks/blob/master/make-util.js#L131-L133

Reply:
Done